util/src/main/resources/tech/pegasys/teku/util/config/swift.yaml

```diff
@@ -0,0 +1,156 @@
+# Minimal preset
```

Comment:
Might be helpful to add some comments that mark which fields differ from minimal, and some general comments at the top about what this config is for and the fact that it is based on minimal

Reply:
Makes sense.

Reply:
Done.